lib/Renard/Curie/Component/PageDrawingArea.pm

```diff
@@ -276,6 +277,42 @@ callback on_key_press_event_cb($window, $event, $self) {
 	}
 }
 
+=method setup_scroll_bindings
+
+  method setup_scroll_bindings()
+
+Sets up the signals to capture scroll events on this component.
+
+=cut
+method setup_scroll_bindings() {
+	$self->scrolled_window->signal_connect(
+		'scroll-event' => \&on_scroll_event_cb, $self );
+}
+
+=callback on_scroll_event_cb
+
+  callback on_scroll_event_cb($window, $event, $self)
+
+Callback that responds to specific scroll events and dispatches the associated
+handlers.
+
+=cut
+callback on_scroll_event_cb($window, $event, $self) {
```

Comment:
Change the function type to `callback`. This is a new type I created just for doc purposes. Make it clearer where it is used.

There is a corresponding POD directive `=callback`.